staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -1030,13 +1030,14 @@ func (e *Store) DeleteCollection(ctx context.Context, deleteValidation rest.Vali
 					errs <- err
 					return
 				}
-				if _, _, err := e.Delete(ctx, accessor.GetName(), deleteValidation, options); err != nil && !kubeerr.IsNotFound(err) {
+				delOptions := options.DeepCopy()
+				if _, _, err := e.Delete(ctx, accessor.GetName(), deleteValidation, delOptions); err != nil && !kubeerr.IsNotFound(err) {
```

Comment:
can  this be simplified to just pass `options.DeepCopy()` to `e.Delete()`? I'm not seeing what the closure parameter is getting us